app/controllers/idv/personal_key_controller.rb

```diff
@@ -2,16 +2,23 @@ module Idv
   class PersonalKeyController < ApplicationController
     include Idv::AvailabilityConcern
     include IdvSession
+    include IdvStepConcern
     include StepIndicatorConcern
     include SecureHeadersConcern
     include FraudReviewConcern
```

Comment:
IdvStepConcern includes IdvSession and FraudReviewConcern and :confirm_two_factor_authenticated, so those can be removed.